lib/constable/models/announcement.ex

```diff
@@ -39,32 +39,41 @@ defmodule Constable.Announcement do
   end
 
   def with_announcement_list_assocs(query \\ __MODULE__) do
-    from q in query, preload: [
-      :user,
-      interests: ^Interest.ordered_by_name,
-      comments: ^newest_comments_first(),
-      comments: :user
-    ]
+    from q in query,
+      preload: [
+        :user,
+        interests: ^Interest.ordered_by_name(),
+        comments: ^newest_comments_first(),
+        comments: :user
+      ]
+  end
+
+  def for_user(query \\ __MODULE__, user_id) do
```

Comment:
This was a new function added to this file. The rest of the changes are due to auto-formatting.